### ISSUE TITLE
UserWarning: masked_fill_ received a mask with dtype torch.uint8

### ISSUE THREAD

**Issue by soerenetler** [jiesutd/NCRFpp#137]
Hi,
I am running the sample training and decoding configuration with the sample training data. I get this warning cluttering my command line (but training and decoding still seams to work). I use the newest PyTorch version 1.2.0. Can this be fixed easily? (I am not very familiar with PyTorch)

Thank you for your help in advance :)

```
/pytorch/aten/src/ATen/native/LegacyDefinitions.cpp:14: UserWarning: masked_fill_ received a mask with dtype torch.uint8, this behavior is now deprecated,please use a mask with dtype torch.bool instead.
```

**Comment by stevehamwu** [comment]
> Hi,
> I am running the sample training and decoding configuration with the sample training data. I get this warning cluttering my command line (but training and decoding still seams to work). I use the newest PyTorch version 1.2.0. Can this be fixed easily? (I am not very familiar with PyTorch)
> 
> Thank you for your help in advance :)
> 
> ```
> /pytorch/aten/src/ATen/native/LegacyDefinitions.cpp:14: UserWarning: masked_fill_ received a mask with dtype torch.uint8, this behavior is now deprecated,please use a mask with dtype torch.bool instead.
> ```

The same problem. Fixed it by changing every
"mask = blahblahblah.byte()"
to
"mask = blahblahblah.bool()"
in three files: main.py, main_parse.py and model/crf.py

**Comment by jiesutd** [comment]
Thanks for the update!
